Automattic/wp-calypso
# Add VideoPress flow to "tailored signup" flow to skip plugins upsell

#### Proposed Changes

* Add a separate "Tailored Signup" helper method to skip plugins upsell

I have not updated all `isNewsletterOrLinkInBioFlow` calls since they don't all apply to hiding the "plugins upsell" page, and since the VideoPress flow is a different flow from the `/create` flows, the flow doesn't work if the VideoPress flowname is included in all of those checks.

@Automattic/shilling if there is a better name for this method I'm happy to make changes. I am just going off the clues I had in the existing code. `skipPluginUpsellFlow` might be a more clear, deliberate name?

#### Testing Instructions



* start the VideoPress signup flow at `/start/videopress`
* Complete the flow and payment
* After payment, you should be sent to the site editor and NOT see the plugins upsell page
* Start a regular signup flow at `/start`, Choose the Premium plan, you should see the plugins upsell after purchase
* Start a link in bio flow at `/setup/intro?flow=link-in-bio`, you should not see the plugins upsell after purchase

#### Pre-merge Checklist






Related to # 1283-gh-Automattic/greenhouse
